.buildkite/longruns/pipeline.yml

```diff
@@ -55,14 +55,7 @@ steps:
   - group: "Coupler integration and conservation tests"
 
     steps:
-      - label: "ClimaAtmos standalone: aquaplanet, allsky rad, TV insol, 0M"
```

Comment:
Removing this is fine - this test has served its purpose through early ClimaCoupler development.